Makefile

```diff
@@ -0,0 +1,17 @@
+# Makefile based on https://www.cs.colby.edu/maxwell/courses/tutorials/maketutor/
+
+CC=cc
```

Comment:
why does cc need to be a variable? is it common for the way you compile to change over time so that the variable is useful?

Reply:
`CC` and `CFLAGS` are special variables for `make`, see [this tutorial](https://www.cs.colby.edu/maxwell/courses/tutorials/maketutor/) under "Makefile 2".

Reply:
yes it is common to change it!